### DISCUSSION TITLE
Possibility to sort tags in backoffice

### DISCUSSION THREAD

**Discussion by Viktor-Abrahamsson** [umbraco/Umbraco-CMS#13430] [Features and ideas]
Hello!

Umbraco.Tags is a powerful tool when used correctly, however, it would be even better if there was a possibility to easily sort the tags in the backoffice for the editor.

If a editor wants to rearrange the order where the tags are displayed on the item, they need to remove all the tags and then create them in the wanted order. It would be easier for them to just drag en drop the existing tags in the desired order. 

![image](https://user-images.githubusercontent.com/107833485/202427607-973a27df-03f7-41da-b884-c7f8a8ed972d.png)

**Comment by lassefredslund** [comment]
Hi @Viktor-Abrahamsson. Thank you for the suggestion, which I think makes sense and would be nice. I've set it up for grabs.
